addOns/ascanrules/src/main/java/org/zaproxy/zap/extension/ascanrules/CommandInjectionScanRule.java

```diff
@@ -551,6 +551,11 @@ private boolean testCommandInjection(
                             .setAttack(paramValue)
                             .setEvidence(matcher.group())
                             .setMessage(msg)
+                            .setOtherInfo(
+                                    String.format(
+                                            "The scanner was able to retrieve the content of a file or command "
```

Comment:
- These should be moved to `Messages.properties` file.
 - Be good to assert these in the tests.

Reply:
Tried really hard here.... Let me know if it's OK (I couldn't figure out the [0] based interpolation)